packages/contracts-por/test/utils/hardhatPatches.ts

```diff
@@ -1,10 +1,11 @@
+// @ts-ignore
```

Comment:
Including hardhat.config.ts in files that are looked at by the TypeScript compiler solves the problem with import. Source: https://github.com/NomicFoundation/hardhat/issues/1627#issuecomment-872713053.

Reply:
Done, thanks!